src/common/Settings.hpp

```diff
@@ -147,6 +147,7 @@ class Settings
 
 	struct s_input_port {
 		int Type;
+		int SlotType[2];
```

Comment:
```suggestion
		int SlotType[XBOX_CTRL_NUM_SLOTS];
```
If not able to do above method, please add `XBOX_CTRL_NUM_SLOTS` in a comment on this line.